src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -363,4 +363,7 @@
   <data name="BreakIntoDebuggerCommandKey" xml:space="preserve">
     <value>Break into the debugger</value>
   </data>
-</root>
+  <data name="OpenSettingsUICommandKey" xml:space="preserve">
+    <value>Open Settings UI</value>

```

Comment:
```suggestion
    <value>Open Settings...</value>
```

I went with `...` because of the convention that things that pop up dialogs or open new windows should be `...`ed